core/search.go

```diff
@@ -35,51 +36,57 @@ func NewSearchRequests(r repo.Repo, cli *rpc.Client) *SearchRequests {
 	}
 }
 
+// Result struct
+type Result struct {
```

Comment:
Would recommend calling this SearchResult, both to make it clear in other places what role this struct serves, and also to bring it closer in line with SearchParams, and other structs like LookupParams / LookupResults in core/datasets.go